### DISCUSSION TITLE
建议将一些开发者加入cooperator

### DISCUSSION THREAD

**Discussion by xiehuc** [rime/home#1364] [General]
squirrel 上的issue和MR太多了都没有处理。
要是作者精力不够的话可以把一些人加到协作者里面，让他们能帮助继续管理项目。

总是有年轻人精力更多的，也很有意愿。

**Comment by MokOopsing** [comment]
你搞清楚协作的工作方式了吗？
搞开源不是开公司，由研发经理分配开发任务，而是由开发者自发地的去修改新增代码，然后提交pr，管理员检视无误后合入。不是把你加入到“cooperator”，你就能解决issue啊，mr/pr也不是无脑合入

**Reply by xiehuc** [reply]
![image](https://github.com/rime/home/assets/571878/9827c2a8-b09c-4fd7-8eec-d46e15a6e193)
管理员没有精力检视的时候不找cooperator么。

https://github.com/LEOYoon-Tsaw/squirrel/releases
LEOYoon的fork都更新0.17.2了，主干的还是0.16的，主干的不合么。

以后大家都用LEOYoon的fork了项目不分裂么。

我TM哪句说了要把我加cooperator了。应该把@LEOYoon-Tsaw加入不对么。

**Reply by MokOopsing** [reply]
我意思不是把谁加进来，谁就能干活

你说的这位本来就是吧，他有权限

![image](https://github.com/rime/home/assets/39441028/e6cea144-8a27-4b43-9f06-062d883d7e34)

火气不要那么大，这里的“你”并不是一定指的你

**Reply by fxliang** [reply]
因为很多个pr的groverlynn和LEOYoon-Tsaw就开发方向上面存在较大的分歧，所以LEOYoon-Tsaw虽然有权限也没有去动主仓的代码吧

**Comment by xiehuc** [comment]
> 我意思不是把谁加进来，谁就能干活
> 
> 你说的这位本来就是吧，他有权限
> 
> ![image](https://user-images.githubusercontent.com/39441028/266515734-e6cea144-8a27-4b43-9f06-062d883d7e34.png)
> 
> 火气不要那么大，这里的“你”并不是一定指的你

明白了，thanks。
